bokehjs/src/coffee/tool/inspectors/hover_tool.coffee

```diff
@@ -130,6 +130,9 @@ class HoverToolView extends InspectTool.View
       vars = {index: i, x: x, y: y, vx: vx, vy: vy, sx: sx, sy: sy, data_x: data_x, data_y: data_y, rx:rx, ry:ry}
       tooltip.add(rx, ry, @_render_tooltips(ds, i, vars))
 
+      vars['renderer'] = renderer
+      @mget('callback')?.execute(vars)
```

Comment:
I don't think there's any reason to say it shouldn't happen frequently. It can happen any time the glyphs overlap, which is actually fairly common I think. I'd much rather have a predictable "one callback call per mousemove event" with the callback getting a list of indices.

In fact I think it has to be this way, what if you want to do some coordination or aggregation if there are multiple hits? It is impossible to do if you pass each index individually.